src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -724,3 +724,18 @@ def test_wait_for_event_repeated_capabilities(session, channels):
     channels_session = session.channels[channels]
     with channels_session.initiate():
         channels_session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+def test_lcr_attributes(session):
```

Comment:
Delete these tests as per discussion in https://github.com/ni/nimi-python/pull/1740/files#r869326979.

Reply:
I've removed these tests in my latest commit.